quic/s2n-quic/src/tests/setup.rs

```diff
@@ -41,8 +41,14 @@ pub fn tracing_events() -> event::tracing::Subscriber {
             }
         }
 
+        let level = if std::env::var("TRACE").is_ok() {
```

Comment:
nit: The tracing subscriber can handle this logic for you: [default_directive](https://docs.rs/tracing-subscriber/latest/tracing_subscriber/filter/struct.Builder.html#method.with_default_directive)